I can reproduce the linting issue locally on your branch, but not on `develop`:
```
 > make lint                    
lint
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
protocol/messenger_sync_settings.go:6: File is not `goimports`-ed with -local github.com/ethereum/go-ethereum,github.com/status-im/status-go (goimports)
	"go.uber.org/zap"
make: *** [Makefile:315: lint] Error 1
```

--- a/multiaccounts/settings/columns.go
+++ b/multiaccounts/settings/columns.go
@@ -38,9 +38,7 @@ var (
 	Currency = SettingField{
 		reactFieldName: "currency",
 		dBColumnName:   "currency",
-		// TODO resolve issue 6 https://github.com/status-im/status-react/pull/13053#issuecomment-1065179963
 		syncProtobufFactory: &SyncProtobufFactory{
-			inactive:          true, // Remove after issue is resolved
 			fromInterface:     currencyProtobufFactory,
 			fromStruct:        currencyProtobufFactoryStruct,
 			valueFromProtobuf: StringFromSyncProtobuf,
@@ -307,9 +305,7 @@ var (
 		reactFieldName: "stickers/packs-installed",
 		dBColumnName:   "stickers_packs_installed",
 		valueHandler:   JSONBlobHandler,
-		// TODO resolve issue 8 https://github.com/status-im/status-react/pull/13053#issuecomment-1065179963
 		syncProtobufFactory: &SyncProtobufFactory{
-			inactive:          true, // Remove after issue is resolved
 			fromInterface:     stickersPacksInstalledProtobufFactory,
 			fromStruct:        stickersPacksInstalledProtobufFactoryStruct,
 			valueFromProtobuf: BytesFromSyncProtobuf,

--- a/protocol/messenger_sync_settings.go
+++ b/protocol/messenger_sync_settings.go
@@ -2,6 +2,7 @@ package protocol
 
 import (
 	"context"
+	"encoding/json"
 
 	"go.uber.org/zap"
 
@@ -87,6 +88,10 @@ func (m *Messenger) handleSyncSetting(response *MessengerResponse, syncSetting *
 	}
 	if err != nil {
 		return err
+	}
+
+	if v, ok := value.([]byte); ok {
+		value = json.RawMessage(v)
 	}
 
 	response.Settings = append(response.Settings, &settings.SyncSettingField{SettingField: sf, Value: value})

